### ISSUE TITLE
Broken compression of floats below 1e-9

### ISSUE THREAD

**Issue by MarcoForte** [beenotung/compress-json#12]
Hi it seems the float compression does not work for numbers below 1e-9.
All leading zeros are removed.
I ran this snippet on https://npm.runkit.com/compress-json 

```
let data = { 
  tiny: 1.2e-10,
  small: 1.2e-9,
  medium: 0.000123,
  large: 0.01,
}
 
let compressed = compressJson.compress(data) // the result is an 
let reversed = compressJson.decompress(compressed)

reversed = {tiny: 0.12, small: 1.2e-9, medium: 0.000123, large: 0.01}
```

Related to https://github.com/beenotung/compress-json/issues/3#issue-955217845 I presume

**Comment by beenotung** [comment]
This bug is fixed by not reversing the exponential part before encoding the number into base62 (hence `1.2e-10` will not be treated as `1.2e-1`).

The patch is released as compress-json@3.0.0 :rocket: 

Thanks for the bug report, feel free to re-open the issue if the problem is not fixed in your case :)

**Comment by beenotung** [comment]
This is a breaking change, so you may need to migrate the compressed data generated by v2.
